SH4RE/SH4RE/ListingViewModel.swift

```diff
@@ -332,7 +350,7 @@ func acceptRentalRequest(listing_id: String, rental_request_id : String, userId:
         if let document = document, document.exists {
             let data = document.data()!
             let title = data["title"] as? String ?? ""
-            sendRentalStatusMessage(statusMessage: "This request has been accepted",messagePreview: "Request Accepted", userId: userId, renterId: renterId, title: title)
+            sendRentalStatusMessage(statusMessage: "This request has been accepted",messagePreview: "Request Accepted", userId: userId, renterId: renterId, title: title, isReviewRequest: false)
```

Comment:
does it make sense to have a default of isReviewRequest=false and then only set it to true on the call for the review request?

Reply:
It just make it easier for me to keep track of when calling it